### PR TITLE
Sample commands

### PR DESCRIPTION
**Pull request by dprestegard**
Simple 1 pass and 2 pass encoding examples

### CODE REVIEW

File: Docs/svt-av1_encoder_user_guide.md [hunk 1]
```diff
@@ -99,7 +101,24 @@ For example, the following command encodes 100 frames of the YUV video sequence
 
 It should be noted that not all the encoder parameters present in the `Sample.cfg` can be changed using the command line.
 
-#### List of all configuration parameters
+### Sample command lines
+
+Here are some sample encode command lines
+
+#### 1 pass fixed QP at maximum speed from 24fps yuv 1920x1080 input
+`svtav1encapp -i input.yuv -w 1920 -h 1080 -fps 24 -rc 0 -q 30 -enc-mode 8 -b output.ivf`
```
Comment (JackZhouVCD) [inline Docs/svt-av1_encoder_user_guide.md]: I think you mean `SvtAv1EncApp` for all of these

Comment (dprestegard) [reply Docs/svt-av1_encoder_user_guide.md]: Ah, good point. You can tell I'm a windows guy, huh? :)

File: Docs/svt-av1_encoder_user_guide.md [hunk 2]
```diff
@@ -99,7 +101,24 @@ For example, the following command encodes 100 frames of the YUV video sequence
...
+
+#### 1 pass VBR 10000 Kbps at medium speed from 24fps yuv 1920x1080 input
+`SvtAv1EncApp -i input.yuv -w 1920 -h 1080 -fps 24 -rc 2 -tbr 10000000 -enc-mode 5 -b output.ivf`
+
+#### 2 pass VBR 5000 Kbps at slow speed from 24fps yuv 1920x1080 input
+Note how this requires two separate executions, one writing a stats file and one reading a stats file
+
+`SvtAv1EncApp -i input.yuv -w 1920 -h 1080 -fps 24 -rc 2 -tbr 5000000 -enc-mode-2p 4 -output-stat-file stats.file -b output.ivf`
+
+`SvtAv1EncApp -i input.yuv -w 1920 -h 1080 -fps 24 -rc 2 -tbr 5000000 -enc-mode 1 -input-stat-file stats.file -b output.ivf`
```
Comment (JackZhouVCD) [inline Docs/svt-av1_encoder_user_guide.md]: Eventually we are going to merge the two commands into one for running 2 pass (see #746), so this bit may need to change to accommodate that when it happens.

Comment (dprestegard) [reply Docs/svt-av1_encoder_user_guide.md]: Great, happy to refactor once that's merged.

File: Docs/svt-av1_encoder_user_guide.md (hunk 1 above)
Comment (dprestegard) [reply Docs/svt-av1_encoder_user_guide.md]: Fixed

**Pull request merged**